lib/Client.js

```diff
@@ -1274,6 +1274,16 @@ class Client extends EventEmitter {
         return this.requestHandler.request("POST", Endpoints.CHANNEL_CROSSPOST(channelID, messageID), true).then((message) => new Message(message, this));
     }
 
+    /**
+     * Follow a NewsChannel
```

Comment:
Permissions needed to do this should be listed here as well

Reply:
See above.

Reply:
ccc